tdrs-backend/tdpservice/parsers/schema_defs/tanf/t1.py

```diff
@@ -209,7 +207,7 @@
                     endIndex=31,
                     required=True,
                     validators=[
-                        validators.oneOf([1, 2]),
+                        validators.matches(1)
```

Comment:
@elipe17 can we drop 3 as a valid possible option for item # 13 (`RECEIVES_SUB_HOUSING`)